libraries/chain/protocol/transaction.cpp

```diff
@@ -359,6 +359,42 @@ set<public_key_type> signed_transaction::get_required_signatures(
    return result;
 }
 
+void signed_transaction::get_required_signatures(
+	const chain_id_type& chain_id,
+	const flat_set<public_key_type>& available_keys,
+	const std::function<const authority*(account_id_type)>& get_active,
+	const std::function<const authority*(account_id_type)>& get_owner,
+	set<public_key_type> &conti_use_reult,
+	uint32_t max_recursion /*= GRAPHENE_MAX_SIG_CHECK_DEPTH*/,
+	uint32_t need_authority_type /*= 0*/ /*0 no need ,1 need other*/
```

Comment:
It would be simpler to make `conti_use_result` a pointer instead of a reference, and use a null pointer to indicate that you don't want the keys.

Reply:
thanks，this is a good idea